### PR TITLE
Added L6 Th+ SST

### LINKED ISSUE #1100: [NTR] L6 Th+ SST/PV cell
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
L6 tyrosine hydroxylase positive Sst expressing Pvalb-like GABAergic cortical interneuron (Mus musculus)

**Synonyms**
L6 Th+ SST/PV (Mus)

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A Sst GABAergic cortical interneuron found in L6 that expresses tyrosine hydroxylase and is transcriptomically related to Pvalb interneurons. L6 Th+ SST/PV cells have mostly local axonal arborization within L6. (PMID:22357664)

NOTE: these cells do not express Pvalb according to the fireplot in https://github.com/obophenotype/brain_data_standards_ontologies/issues/89 

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
Interneuron

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
Cerebral Cortex

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**
@dosumis not sure if this is ok as the PV part seems transcriptomically related rather than expressing (at least with the dataset that the fireplot is built on)

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #1100

### CODE REVIEW

**Review by patrick-lloyd-ray — APPROVED** [review]

File: src/ontology/cl-edit.owl
Comment (shawntanzk) [inline src/ontology/cl-edit.owl]: @dosumis just to double check before merging anything, is "transcriptomically related" ok for CL? Thanks

**Review by dosumis — COMMENTED** [review]
Needs more discussion

**Comment by shawntanzk** [comment]
Need to refresh PR import to bring in TH (not sure why its not there now).

**Comment by shawntanzk** [comment]
@dosumis will not merge until you approve - thanks

**Pull request merged**
